internal/test/framework/windowsvm.go

```diff
@@ -39,9 +39,28 @@ type windowsVM struct {
 	sshClient *ssh.Client
 	// winrmClient to access the Windows VM created
 	winrmClient *winrm.Client
+	// windowsVMLabels is a map of test specific labels for the Windows VM
+	*windowsVMLabels
```

Comment:
This should not be here. What you need to do is create a `testWindowsVM` struct and embed the bool in there.
```
type testWindowsVM struct {
    windowsVM
    buildWMCB bool
}
```